plugins/plugin-json/che-plugin-json-server/src/main/java/org/eclipse/che/plugin/json/languageserver/JsonLanguageServerLauncher.java

```diff
@@ -58,8 +49,7 @@ public boolean isAbleToLaunch() {
     }
 
     protected LanguageServer connectToLanguageServer(final Process languageServerProcess, LanguageClient client) {
-        Launcher<LanguageServer> launcher = Launcher.createLauncher(client, LanguageServer.class,
```

Comment:
That would mean that I have to create an issue and open a PR just for a formatting change? I politely disagree here. My favorite solution would be to automate formatting completely, then formatting changes would never occur.